### PR TITLE
Fixed bug & tests

### PR DESCRIPTION
**Pull request by hacheka**
Fixes a corner case in the SQL.getBounds function: when the API returns no bounds, neither the promise was resolved nor the callback was called.

CR @alonsogarciapablo 
CC @xavijam

### CODE REVIEW

File: src/api/sql.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ var $ = require('jquery');
 var Mustache = require('mustache');
 var Promise = require('./promise');
 
+var errorMessages = {
```
Comment (alonsogarciapablo) [inline src/api/sql.js]: Something I learnt recently... if we use something like `NO_BOUNDS_ERROR_MESSAGE` here (instead of an object), it's easier for us (well, the linter) to catch unused variables.

Comment (hacheka) [inline src/api/sql.js]: I did it this way to have all error messages (so far we only have one) in an object instead of having a bunch of error messages each in one variable.

Changing it.
